services/src/test/scala/org/apache/gearpump/services/TestCluster.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.gearpump.services
+
+import akka.actor.ActorRef
+import org.apache.gearpump.cluster.TestUtil
+import org.apache.gearpump.cluster.TestUtil.MiniCluster
+import org.apache.gearpump.streaming.StreamingTestUtil
+import org.apache.gearpump.util.LogUtil
+
+object TestCluster {
+  private val LOG = LogUtil.getLogger(getClass)
+
+  lazy val cluster: MiniCluster = TestUtil.startMiniCluster
+  lazy val master: ActorRef = {
+    LOG.info("Accessing master of TestCluster...")
+    StreamingTestUtil.startAppMaster(cluster, 0)
+    cluster.mockMaster
+  }
+}
```

Comment:
There may exist a bug in akka cluster, 
https://github.com/akka/akka/issues/17253

It will cause the akka cluster actor system failed to start. The singleton is a work-around to avoid frequent start/stop of akka cluster.